scripts/get-sampleapp-url.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eE
+
+KUBECONFIG=""
+DEBUGFILE="/tmp/getsampleappurl.log"
+if [[ -e "${DEBUGFILE}" ]]; then
+    rm -f "${DEBUGFILE}"
+fi
+
+# jq reads from stdin
+function parse_input() {
+  eval "$(jq -r '@sh "export KUBECONFIG=\(.KUBECONFIG) APPNAMESPACE=\(.APPNAMESPACE) APPNAME=\(.APPNAME)"')"
```

Comment:
this means we have a dependency on `jq` - is it possible to remove this dependency?

Reply:
I need to parse the json, so the alternative is to use python instead of bash